awx/main/tasks.py

```diff
@@ -354,6 +355,22 @@ def send_notifications(notification_list, job_id=None):
 
 @task(queue=get_local_queuename)
 def gather_analytics():
+    def _gather_and_ship(subset, since=None, until=None):
```

Comment:
again, can be:

```
_gather_and_ship(subset, since, until):
```

since the time window must be always defined

Reply:
done